src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1247,9 +1243,7 @@ namespace Psychrometrics {
 
 	inline
 	Real64
-	CPHW(
-		Real64 const Temperature // unused1208
-	)
+	CPHW()
```

Comment:
yes,  constant is fine.  all "real" plant loop models use temperature-dependent calls to glycol functions.  I once converted most all uses of CPCW and CPHW to general fluid properties.  the remaining applications (stand-alone, sizing) can use a constant.